config.defaults.json

```diff
@@ -22,7 +22,41 @@
     "telegram": {
       "enabled": false,
       "botToken": "See README",
-      "chats": []
+      "chats": [],
+      "commands": {
```

Comment:
I don't see what it adds to make these configurable.
Since I like to keep things simple, I would remove this config.

Reply:
I hesitated with that. I use this part for the getCommands function used to set commands list in Telegram chat but I can move it into telegram-bot.js

Reply:
Please do

Reply:
done ;)